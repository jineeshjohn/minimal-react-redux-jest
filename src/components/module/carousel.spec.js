import React from 'react';
import { shallow } from 'enzyme';
import Carousel from './carousel';

describe('Carousel specs', () => {
	const defaultProps = {
		title: 'this is test title'
	};

	const ShallowComponent = props => shallow(<Carousel {...props} />);

	it('should shallow render as expected', () => {
		const component = ShallowComponent(defaultProps);
		expect(component).toMatchSnapshot();
	});
});
