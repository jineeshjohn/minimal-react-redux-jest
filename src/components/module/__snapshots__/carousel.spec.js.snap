// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel specs should shallow render as expected 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Carousel
    title="this is test title"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        " Testing components Carousel blue!! ",
        "this is test title",
        " ",
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      " Testing components Carousel blue!! ",
      "this is test title",
      " ",
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          " Testing components Carousel blue!! ",
          "this is test title",
          " ",
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        " Testing components Carousel blue!! ",
        "this is test title",
        " ",
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
